module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,169 @@
+import "./MovieForm.css";
+interface MovieFormProps {
+  title: string;
+  movie?: Movie;
+  closeCallback: () => void;
+}
+
+export const MovieForm = ({ title, movie, closeCallback }: MovieFormProps) => {
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    console.log("HandleSubmit");
+  }
+  const handleClose = () => closeCallback();
+  return (
+    <div className="formSettings bg-custom_dark_gray  min-w-full">
+      <div className="flex text-white justify-end mb-1">
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          fill="none"
+          viewBox="0 0 24 24"
+          strokeWidth={1.5}
+          stroke="currentColor"
+          className="w-6 h-6"
+          onClick={() => handleClose()}
+        >
+          <path
+            strokeLinecap="round"
+            strokeLinejoin="round"
+            d="M6 18 18 6M6 6l12 12"
+          />
+        </svg>
+      </div>
+      <h1 className="ml-5 mb-2 text-white text-2xl font-Montserrat">{title}</h1>
+      <form onSubmit={(e) => handleSubmit(e)}>
```

Comment:
If you pass all the parameters to the event handler you can use this shorter version
```suggestion
      <form onSubmit={handleSubmit}>
```